twake/backend/node/src/core/platform/framework/decorators/realtime/saved.ts

```diff
@@ -8,20 +8,20 @@ import { eventBus } from "../../../services/realtime/bus";
  * @param path the path to push the notification to
  * @param resourcePath the path of the resource itself
  */
-export function RealtimeUpdated<T>(path: string | PathResolver<T>, resourcePath?: string | PathResolver<T>): MethodDecorator {
+export function RealtimeSaved<T>(path: string | PathResolver<T>, resourcePath?: string | PathResolver<T>): MethodDecorator {
   // eslint-disable-next-line @typescript-eslint/ban-types
-  return function(target: Object, propertyKey: string, descriptor: PropertyDescriptor ): void {
+  return function (target: Object, propertyKey: string, descriptor: PropertyDescriptor): void {
```

Comment:
`function (target: Object,...)` or ` function (target: object,...)` ?
I think you should use the second one in typescript.

Reply:
We need to define the rules for this

Reply:
Simply install prettier on VSCode and copy the .prettierrc from twake/frontend/.prettierrc
Then enable prettify on save.